gas/testsuite/gas/riscv/cv-march-rv32i-xcorev.s

```diff
@@ -0,0 +1,3 @@
+# Testing xcorev march option works for all
```

Comment:
Should this refer to `xcorevhwlp`, not `xcorev`

Reply:
No, this test is to test that all of the CORE-V extensions get called by xcorev. We will put an instruction from each extension. I will just update the .d file.